scss/modules/_site-header.scss

```diff
@@ -247,6 +247,15 @@
   }
 }
 
+.utility-nav__search {
+  padding-right: u(2rem);
+  width: u(30rem);
+
+  .tt-menu {
+    z-index: $z-max !important;
```

Comment:
I'm going to agree with Hound here, is there a reason why this is needed?  Okay if there is, just want to make sure we're being deliberate with it!